### PR TITLE
Update RBAC docs with new nav properties

### PR DESCRIPTION
**Pull request by VinceSmith**
Added content for navigation properties in roleAssignments.

### CODE REVIEW

File: api-reference/beta/api/unifiedroleassignments-list.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+title: "List roleAssignments by principal"
+description: "List roleAssignment objects by principal."
+localization_priority: Normal
+author: "davidmu1"
+ms.prod: "microsoft-identity-platform"
+doc_type: "apiPageType"
+---
+# List principal
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+List roleAssignment objects by principal (for example, a given user).
```
Comment (dkershaw10) [inline api-reference/beta/api/unifiedroleassignments-list.md]: Maybe I'm totally misunderstanding this but, I though this is just the (full) principal that is assigned to a role as part of a roleAssignment.  In the unifiedRoleAssignment definition, it says "Get a principal object collection."

So should this read: "List the principals (e.g. users) for this role assignment.

But I'm not more confused.  Is this a collection or a single navigation?  The associated (existing property) - **principalId** is not a collection.  The navigation is named in the singular, but is defined as a collection.  Which is it?

File: api-reference/beta/api/unifiedroleassignments-list.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
+---
+title: "List principals for a given roleAssignment"
+description: "List principals for a given roleAssignment."
+localization_priority: Normal
+author: "davidmu1"
+ms.prod: "microsoft-identity-platform"
+doc_type: "apiPageType"
+---
+# List principal
```
Comment (dkershaw10) [inline api-reference/beta/api/unifiedroleassignments-list.md]: This is really:
roleAssignment: get principal

File: api-reference/beta/api/unifiedroleassignments-list.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+List principals for a given roleAssignment (for example, users).
```
Comment (dkershaw10) [inline api-reference/beta/api/unifiedroleassignments-list.md]: Get the principal for a given roleAssignment (for example, user, group or servicePrincipal).

File: api-reference/beta/api/unifiedroleassignments-list.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+## Optional query parameters
+This method supports some of the OData query parameters to help customize the response. For general information, see [OData query parameters](/graph/query-parameters).
+## Request headers
+| Name      |Description|
+|:----------|:----------|
+| Authorization | Bearer {token} |
+## Request body
+Do not supply a request body for this method.
+## Response
+If successful, this method returns a `200 OK` response code and a collection of [directoryObject](../resources/directoryobject.md) objects in the response body.
```
Comment (dkershaw10) [inline api-reference/beta/api/unifiedroleassignments-list.md]: It's not returning a collection...

File: api-reference/beta/api/unifiedroleassignments-list.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  "truncated": true,
+  "@odata.type": "microsoft.graph.directoryObject",
+  "isCollection": true
+} -->
+```http
+HTTP/1.1 200 OK
+Content-type: application/json
+{
+    "@odata.context": "https://graph.microsoft.com/beta/$metadata#directoryObjects",
+    "value": [
```
Comment (dkershaw10) [inline api-reference/beta/api/unifiedroleassignments-list.md]: It's not a collection...

**Comment by dkershaw10** [comment]
@VinceSmith Can you get this completed please?  The service has already been updated with the new navigation properties...

**Comment by Lauragra** [comment]
@VinceSmith , please incorporate feedback on this PR and address the validation errors (looks like broken links).

**Comment by lramosvea** [comment]
In the live metadata, I can see the roleAssignments and roleDefinitions navigation properties for the rbacapplication resource but there are other nav properties which have not been documented either.

**Pull request closed without merging**